apps/quick-dapp/src/App.css

```diff
@@ -0,0 +1,146 @@
+/* You can add global styles to this file, and also import other style files */
```

Comment:
Please use as little manual styling as possible.
- Bootstrap classes should be prioritized over custom styles
- Flexbox, instead of manual calculations, that allows to adjust dynamically based on the available space,